words/words.md

```diff
@@ -358,117 +358,117 @@ But wait... haven't Taiwan and South Korea *already* contained COVID-19? For 4 w
 
 How?
 
-###Scenario 4: Test, Trace, Isolate
+###Scénario 4 : Test, Suivi, Isolement
 
-*"Sure, we \*could've\* done what Taiwan & South Korea did at the start, but it's too late now. We missed the start."*
+*« Bien sûr, on \*pourrait\* avoir fait comme Taïwan et la Corée du Sud ont fait au début, mais c'est désormais trop tard. On a loupé le départ. »*
```

Comment:
on pourrait avoir fait --> nous aurions pu faire ?
au moins "on aurait pu faire" imho

on perd la répétition , ...at the **start**, {too late} --> we missed the **start**.  c'est dommage imho.
--> nous aurions pu faire comme taiwan et la coree  du sud dès le départ, mais c'est trop tard. Nous avons loupé le départ.   ?